src/main/cms/cms_types.h

```diff
@@ -53,29 +53,31 @@ typedef enum
 
 typedef long (*CMSEntryFuncPtr)(displayPort_t *displayPort, const void *ptr);
 
+typedef uint8_t OSD_flags_t;
+
 typedef struct
 {
     const char *text;
     const OSD_MenuElement type;
     const CMSEntryFuncPtr func;
     void *data;
-    uint8_t flags;
+    OSD_flags_t flags;
 } OSD_Entry;
 
 // Bits in flags
 #define PRINT_VALUE    0x01  // Value has been changed, need to redraw
 #define PRINT_LABEL    0x02  // Text label should be printed
 #define DYNAMIC        0x04  // Value should be updated dynamically
 
-#define IS_PRINTVALUE(p) ((p)->flags & PRINT_VALUE)
-#define SET_PRINTVALUE(p) { (p)->flags |= PRINT_VALUE; }
-#define CLR_PRINTVALUE(p) { (p)->flags &= ~PRINT_VALUE; }
+static inline OSD_flags_t IS_PRINTVALUE(OSD_Entry *p) { return (p->flags & PRINT_VALUE); }
+static inline void SET_PRINTVALUE(OSD_Entry *p) { p->flags |= PRINT_VALUE; }
+static inline void CLR_PRINTVALUE(OSD_Entry *p) { p->flags &= (OSD_flags_t)~PRINT_VALUE; }
 
-#define IS_PRINTLABEL(p) ((p)->flags & PRINT_LABEL)
-#define SET_PRINTLABEL(p) { (p)->flags |= PRINT_LABEL; }
-#define CLR_PRINTLABEL(p) { (p)->flags &= ~PRINT_LABEL; }
+static inline OSD_flags_t IS_PRINTLABEL(OSD_Entry *p) { return (p->flags & PRINT_LABEL); }
+static inline void SET_PRINTLABEL(OSD_Entry *p) { p->flags |= PRINT_LABEL; }
+static inline void CLR_PRINTLABEL(OSD_Entry *p) { p->flags &= (OSD_flags_t)~PRINT_LABEL; }
```

Comment:
Well, almost. *VALUE vs *LABEL. 
Maybe you see a smarter way, fewer parametrized functions instead of C&P&M (Copy&Paste&Modify) repetitions.